seahub/templates/snippets/reset_repo_password.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'email_base.html' %}
+
+{% load i18n %}
+
+{% block email_con %}
+
+<p style="color:#121214;font-size:14px;">{% trans "Hi," %}</p>
+
+<p style="font-size:14px;color:#434144;">
+{% blocktrans %}Password of library {{ repo_name }} has been reset. Now it is {{ password }}. Please change it as soon as possible.{% endblocktrans %}
```

Comment:
The new password of library {{ repo_name }} is {{password}}. Please change it as soon as possible.